iac-extensions/terraform/src/test/resources/checks/GCP/ComputeInstanceSshKeysCheck/ssh_keys.tf

```diff
@@ -0,0 +1,63 @@
+resource "google_compute_instance" "example1" { # Noncompliant {{Omitting metadata.block-project-ssh-keys enables project-wide SSH keys. Make sure it is safe here.}}
+#        ^^^^^^^^^^^^^^^^^^^^^^^^^
+}
+
+resource "google_compute_instance" "example2" {
+  metadata = true # Noncompliant {{metadata of type Object expected.}}
+# ^^^^^^^^^^^^^^^
```

Comment:
I would not report here because it's invalid semantic. This rule is not responsible to report invalid semantic